gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,6 @@
+#Thu Jan 14 01:25:24 KST 2021
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

bin -> all 로 프로퍼티 변경
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```